config/settings/development.yml

```diff
@@ -15,3 +15,6 @@ betamocks:
 
 account:
   enabled: true
+
+locators:
+  mock_gis: true
```

Comment:
This setting and the reference to it in the GIS client configurations are all that is required to make `https://dev-api.va.gov` use the mock data, correct?